src/org/feedhenry/Utils.groovy

```diff
@@ -36,3 +38,9 @@ def getArtifactsDir(name) {
 def gitRepoIsDirty(untrackedFiles='no') {
     return sh(returnStdout: true, script: "git status --porcelain --untracked-files=${untrackedFiles}").trim()
 }
+
+static Closure getDate() {
```

Comment:
Also, not sure if this is a good name or not for this method. Suggestions welcome!